### DISCUSSION TITLE
Unable to import the pytorch3d package

### DISCUSSION THREAD

**Discussion by utkarsh0702** [facebookresearch/pytorch3d#1697] [Q&A]
I have tried installing pytorch3d on my system. It successfully installed but when I try to import it, it shows the below error.
![Screenshot 2023-12-11 174748](https://github.com/facebookresearch/pytorch3d/assets/35002267/78c2031f-73aa-4221-8578-97e56ae37ef3)

**Comment by bottler** [comment]
PyTorch3D is not installed correctly. For example the version of cuda in your environment is not the same as the version for which pytorch3d was built. How did you install?

**Reply by utkarsh0702** [reply]
I installed it using pip. The version of Cuda was 116 and Python was 3.8

**Reply by bottler** [reply]
What exact commands did you run to install? Did you follow our INSTALL.md?

**Reply by utkarsh0702** [reply]
Yes, I went through the INSTALL.md file. As I have a Linux docker container I made Cuda, Python and PyTorch versions compatible by referencing that file.

To install PyTorch3d

- First I used:
`$pip install pytorch3d`
Then I found out it was an older version

- Afterwards, I used the code:

```
import sys
import torch
pyt_version_str=torch.__version__.split("+")[0].replace(".", "")
version_str="".join([
    f"py3{sys.version_info.minor}_cu",
    torch.version.cuda.replace(".",""),
    f"_pyt{pyt_version_str}"
])
print(version_str)

!pip install --no-index --no-cache-dir pytorch3d -f https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/{version_str}/download.html 
```
